.gitlab-ci.yml

```diff
@@ -272,6 +302,45 @@ test_macos11_py37_mil:
       - coremltools/converters/mil/mil/**/*.{py}
       - coremltools/converters/mil/backend/**/*.{py}
 
+#########################################################################
+##
+##                macOS - Smoke Test on older versions
+##
+#########################################################################
+
+test_macos11_py38_coremltools_smoke_test:
+  <<: *test_macos_pkg
+  tags:
+    - macos11
+  dependencies:
+    - build_wheel_macos_py38
+  variables:
+    WHEEL_PATH: build/dist/*cp38*10_16*
+    TEST_PACKAGE: coremltools.test.neural_network.test_simple_nn_inference
+    PYTHON: "3.8"
+
+test_macos15_py38_coremltools_smoke_test:
```

Comment:
Prefer to rename this to `macos10_15` (to avoid confusions, this looks newer than 11)

Reply:
Good point, let me do that in a separate PR